core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4004,111 +4004,135 @@ definitions:
         items:
           type: string
 
-  AttributeUpdateOperation:
+  AttributeUpdateOperationArray:
     description: ''
+    type: array
+    items:
+      $ref: '#/definitions/AtttributeUpdateOperation'
+    example: >
+      [
+        {
+          "op": "AddUpdateAttribute",
+          "attributeName": "string",
+          "addUpdateAttribute": "string"
```

Comment:
nit: it might be useful in these examples to make them more example-y, rather than just writing "string" everywhere